app/code/community/Oyst/OneClick/etc/system.xml

```diff
@@ -572,6 +572,26 @@
                     </fields>
                 </reset>
 
+                <new_account translate="label">
+                    <label>Create account for new order</label>
+                    <frontend_type>text</frontend_type>
+                    <sort_order>60</sort_order>
+                    <show_in_default>1</show_in_default>
+                    <show_in_website>1</show_in_website>
+                    <show_in_store>1</show_in_store>
+                    <fields>
+                        <create translate="label">
+                            <label>Create account for new order</label>
```

Comment:
"Create a customer account for a new order"

French "Création d'un compte utilisateur lors d'une nouvelle commande"